test/integration/tests/dependencies.ts

```diff
@@ -23,7 +23,17 @@ describe('e2e-dependencies', function () {
         for (let j = 0; j < contracts.length; j++) {
           const contract = contracts[j];
           const category = addresses[contract].category;
-          if (category === 'External' || category === 'Deprecated') {
+          if (category === 'External') {
```

Comment:
You should make an enum for these categories in the types file and import it here instead of using string literals

Reply:
(and also probably use said enum everywhere else if you can do that without manually editing 500+ lines of code)

Reply:
it already was an enum. Used it here